app/src/main/java/de/tum/in/tumcampusapp/component/ui/tufilm/show_ticket.java

```diff
@@ -14,30 +15,31 @@
 import com.journeyapps.barcodescanner.BarcodeEncoder;
 
 import de.tum.in.tumcampusapp.R;
+import de.tum.in.tumcampusapp.component.other.generic.activity.BaseActivity;
+import de.tum.in.tumcampusapp.utils.Const;
 
-public class show_ticket extends AppCompatActivity {
+public class show_ticket extends BaseActivity {
```

Comment:
Classnames should be uppercase, also activity's classes should have the suffix "Activity". -> ShowTicketActivity

Reply:
Also please in one go fix the filename

Reply:
Got it. I will fix it.